lnclient/cashu/cashu.go

```diff
@@ -375,3 +379,65 @@ func (cs *CashuService) GetSupportedNIP47NotificationTypes() []string {
 func (svc *CashuService) GetPubkey() string {
 	return ""
 }
+
+func (cs *CashuService) GetCustomCommandDefinitions() []lnclient.NodeCommandDef {
+	return []lnclient.NodeCommandDef{
+		{
+			Name:        nodeCommandRestore,
+			Description: "Restore cashu tokens after the wallet had a stuck payment.",
+			Args:        nil,
+		},
+		{
+			Name:        exampleCommandWithArg,
+			Description: "Example command with argument",
+			Args: []lnclient.NodeCommandArgDef{
+				{
+					Name:        "hello",
+					Description: "world",
+				},
+			},
+		},
+	}
+}
+
+func (cs *CashuService) ExecuteCustomCommand(ctx context.Context, command *lnclient.NodeCommandRequest) (*lnclient.NodeCommandResponse, error) {
+	switch command.Name {
+	case nodeCommandRestore:
+		return cs.executeCommandRestore(ctx)
+	case exampleCommandWithArg:
+		if len(command.Args) != 1 {
+			return nil, errors.New("please provide an argument")
+		}
+
+		return &lnclient.NodeCommandResponse{
+			RawJson: []byte("{\"" + command.Args[0].Name + "\": \"" + command.Args[0].Value + "\"}"),
```

Comment:
Here also if we returned an interface instead of RawJson I think it would be simpler?